network-api/networkapi/buyersguide/serializers.py

```diff
@@ -0,0 +1,56 @@
+from rest_framework import serializers
+
+
+class ProductSerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        votes = []
+
+        for range_product_vote in instance.range_product_votes.all():
+            breakdown = {
+                'attribute': 'creepiness',
+                'average': range_product_vote.average,
+                'vote_breakdown': {}
+            }
+
+            for vote_breakdown in range_product_vote.rangevotebreakdown_set.all():
+                breakdown['vote_breakdown'][str(vote_breakdown.bucket)] = vote_breakdown.count
+
+            votes.append(breakdown)
+
+        for boolean_product_vote in instance.boolean_product_votes.all():
+            breakdown = {
+                'attribute': 'confidence',
+                'vote_breakdown': {}
+            }
+
+            for vote_breakdown in boolean_product_vote.booleanvotebreakdown_set.all():
+                breakdown['vote_breakdown'][str(vote_breakdown.bucket)] = vote_breakdown.count
+
+            votes.append(breakdown)
+
+        return {
```

Comment:
The schema has shifted a little bit for Product in my latest PR:

https://github.com/mozilla/foundation.mozilla.org/blob/c704458880858e4da1478bc1f1172868be9463c0/network-api/networkapi/buyersguide/models.py#L42

Reply:
Should I base my PR on yours?

Reply:
Yeah, it's actually landed now.